### DISCUSSION TITLE
bought pro version but still did not receive confirmation and code

### DISCUSSION THREAD

**Discussion by andreynitro** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#904] [General]
Hello just bought pro version hour or so today, but still did not recieve any confirmation and code.. thought that might be an issue with email typo or so... can you guys somehow follow up on this question?

Bought from Canada in CAD..

**Comment by MatthiasKerbl** [comment]
> Hello just bought pro version hour or so today, but still did not recieve any confirmation and code.. thought that might be an issue with email typo or so... can you guys somehow follow up on this question?
> 
> Bought from Canada in CAD..

Hi @andreynitro,

I am sorry you are having trouble with the buying process.
Please take a look at the following steps from our FAQ:

**I Did Not Get The License Code Or Any Receipt Emails After Buying AlDente Pro!**

There are multiple reasons why you did not get your license code after the purchase:

1. Sometimes it takes up to 30 Minutes for the purchase emails to go through.
2. The purchase emails end up in the junk email folder.
3. You had a typo in the email address during your purchase.
4. Failures of the payment networks/providers.

Please contact the support team of our payment provider Paddle at [assist@paddle.com](mailto:assist@paddle.com) and include the email address you used to purchase AlDente Pro. The Paddle support will try to solve your issue as fast as possible.

Cheers,
Matthias
